src/frontend/js/hooks/useBreadcrumbsPlaceholders.tsx

```diff
@@ -5,9 +5,12 @@ import { DashboardBreadcrumbsContext } from 'widgets/Dashboard/contexts/Dashboar
 /**
  * Hook to be used in every component that needs to provide a formatjs value to breadcrumbs's part.
  */
-export const useBreadcrumbsPlaceholders = (data: DashboardBreadcrumbsPlaceholders) => {
+export const useBreadcrumbsPlaceholders = (
+  data: DashboardBreadcrumbsPlaceholders,
+  dependencies: any[],
+) => {
   const context = useContext(DashboardBreadcrumbsContext);
   useEffect(() => {
     context.pushBreadcrumbsPlaceholders(data);
-  }, [data]);
+  }, dependencies);
```

Comment:
What about replacing old `[data]` by `[JSON.stringify(data)]` it would maybe work avoid us to define `dependencies` for each time we use `useBreadcrumbsPlaceholders`, WDYT?